docs/src/operations/requirements.md

```diff
@@ -35,6 +35,7 @@ The hardware recommendations below are provided as a guide.  Operators are encou
   - PCIe Gen3 x4 NVME SSD, or better
   - Accounts: 500GB, or larger. High TBW (Total Bytes Written)
   - Ledger: 1TB or larger. High TBW suggested
+  - Snapshots: 500GB or larger. High TBW suggested
```

Comment:
Looking at mnb data right now, I see:
- ~51 GB full snapshots
- ~4 GB incremental snapshots (peak)

Keeping two fulls + five incrementals would be ~125 GB. If we double that (which accounts and ledger look to be), maybe 256 GB a tighter number here ?

Reply:
Switched to 256GB in d7a3a7aa8e